contributing/TESTING.md

```diff
@@ -402,3 +402,15 @@ gulp e2e
 The task will kick off `gulp build` and then `gulp serve` before running the tests. To skip building the runtime, use `--nobuild`.
 
 [Consult the E2E testing documentation](../build-system/tasks/e2e/README.md) to learn how to create your own end-to-end tests.
+
+## Performance Testing Node Build Tools
+
+You can create flamecharts for any node process used by the build system by leveraging `0x` which is included as a `devDepenendency`.
+
+Here's an example for `gulp dist --closure_concurrency=1`:
+
+```sh
+yarn 0x -o node_modules/.bin/gulp dist --closure_concurrency=1
```

Comment:
What do you think about, as a follow-up, me adding `gulp 0x <whatever>` as a wrapper so `gulp 0x dist --closure_concurrency=1` does the same as above?

Reply:
Sounds like an interesting followup!